build_system/src/test.rs

```diff
@@ -138,6 +138,16 @@ impl TestArg {
                         return Err("Expected an argument after `--features`, found nothing".into())
                     }
                 },
+                "--sysroot" => match args.next() {
```

Comment:
This file already contain an option to build the sysroot [here](https://github.com/antoyo/rustc_codegen_gcc/blob/master/build_system/src/test.rs#L30), so it should not be needed.

Reply:
i will remove this